app/src/main/java/com/codesoom/assignment/utils/HttpMethod.java

```diff
@@ -0,0 +1,13 @@
+package com.codesoom.assignment.utils;
+
+/*
+https://datatracker.ietf.org/doc/html/rfc2616
+ */
+
+public enum HttpMethod {
```

Comment:
기왕 작성한다면 컨벤션을 따라주는 것이 좋겠죠.

```suggestion
/**
 * HTTP 프로토콜 메소드를 정의합니다.
 * @link https://datatracker.ietf.org/doc/html/rfc2616
 */
public enum HttpMethod {
```

Reply:
수정하였습니다. 감사합니다.